integration/integration_test.go

```diff
@@ -957,6 +1148,279 @@ func (s *IntSuite) TestDiscovery(c *check.C) {
 	c.Assert(main.Stop(true), check.IsNil)
 }
 
+// TestExternalClient tests if we can connect to a node in a Teleport
+// cluster. Both normal and recording proxies are tested.
+func (s *IntSuite) TestExternalClient(c *check.C) {
+	// only run this test if we have access to the external ssh binary
+	_, err := exec.LookPath("ssh")
+	if err != nil {
+		return
```

Comment:
use `c.Skip` with message, that will show up in stats and message will be displayed